src/components/background/Background.tsx

```diff
@@ -28,7 +28,7 @@ const BgStyled = styled.div`
 const bubbles = [
   {
     directions: 'to right top',
-    flagUrl: flags.flagDeu,
+    flagUrl: flags.FlagDeu,
```

Comment:
z reguły to się zapisuje z małej litery (`FlagDeu), i wtedy klucze też robi się z małych liter. ale już niech zostanie